CHANGELOG.md

```diff
@@ -4,10 +4,51 @@ Mapbox welcomes participation and contributions from everyone.
 
 # main
 
-# 10.9.0
+# 10.9.0 October 21, 2022
```

Comment:
nit: release date will be automatically added by release train app, and the PR will be automatically updated to both main and release branch.

Reply:
y